src/main/java/com/jmb/springfactory/dao/permission/PermissionMySQLService.java

```diff
@@ -1,10 +1,11 @@
 package com.jmb.springfactory.dao.permission;
 
+import com.jmb.springfactory.dao.GenericMySQLService;
 import com.jmb.springfactory.model.entity.Permission;
 import com.jmb.springfactory.model.enumeration.PermissionsEnum;
+import com.jmb.springfactory.service.GenericService;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.jmb.springfactory.service.GenericService'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)